app/domain/authentication/authn_jwt/validate_generic_status_validations.rb

```diff
@@ -0,0 +1,56 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateGenericStatusValidations ||= CommandClass.new(
```

Comment:
i don't really understand this abstraction. what is a generic validation? what is a non-generic validation?
is there a real value in performing these validations under this class and not under `ValidateStatus`? 

hint - will a reader understand what happens under `ValidateGenericStatusValidations` or will they need to step into the class implementation to understand?

Reply:
The validations in `ValidateGenericStatusValidations` are validations that need to be performed on every authenticator(`validate_webservice_exists`, `validate_user_has_access_to_status_webservice` etc) while `ValidateStatus` validations are specific for jwt authenticator. 
I set them in different files as a preparation for the near future when we will have different authenticators use the same status endpoint and each could use the same  `ValidateGenericStatusValidations` insted of replicating code.

about the class name not being self-explained, im open for name suggestions

Reply:
i would not perform any abstractions for the future. when you get to the point of refactor you can think of the best absteraction for it. i would move these validations into `ValidateStatus` and remove this class.